politeiawww/user/cockroachdb/models.go

```diff
@@ -48,6 +48,18 @@ func (User) TableName() string {
 	return tableUsers
 }
 
+// Session represents a politeiawww user session.
+type Session struct {
+	ID     string    `gorm:"primary_key"` // Unique session id
+	UserID uuid.UUID `gorm:"not null"`    // User UUID (foreign key)
+	Values string    `gorm:"not null"`    // session values (encoded)
```

Comment:
Regarding the comment here, what type of encoding is it?  Hex?

Reply:
It is encoded by `gorilla/securecookie`, see https://github.com/gorilla/securecookie/blob/86450627d8e6b88e00ea41f228489b89d08a40de/securecookie.go#L247

Reply:
Expanded the comment.

Reply:
Looks like its base64 encoded, so I would just add that to the comment.

`// session values (base64 encoded)`

Reply:
Done.